File: OCH_Tao/ABC/B/244.js
const T = [...require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n")[1]];
let ans = [0, 0];
let cnt = 0;
for (const i of T) {
  if (i === "R") {
    cnt++;
  } else {
    if (cnt % 4 === 0) {
      ans[0]++;
    } else if (cnt % 4 === 1) {
      ans[1]--;
    } else if (cnt % 4 === 2) {
      ans[0]--;
    } else {
      ans[1]++;
    }
  }
}
console.log(...ans);

File: OCH_Tao/ABC/B/244.py
N = int(input())
T = list(input())
ans = [0,0]
cnt = 0
for i in T:
  if i=="R":
    cnt+=1
  else:
    if cnt%4==0:
      ans[0]+=1
    elif cnt%4==1:
      ans[1]-=1
    elif cnt%4==2:
      ans[0]-=1
    else:
      ans[1]+=1
print(*ans)